test/led.js

```diff
@@ -361,14 +361,16 @@ exports["Led - PWM (Analog)"] = {
 
     this.led.mode = 1;
     this.led.pulse();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.mode = 1;
     this.led.fade();
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
 
     this.led.strobe();
-    // pre 0.8.13, this would change to 1
+    this.clock.tick(1001);
     test.equal(this.led.mode, 3);
```

Comment:
@rwaldron Since the PWM check and `this.mode` reset is now in `ledWrite` (since it's controller-specific), the mode doesn't get reset until the first write has fired.

That said the default delay is actually 10ms, so they should be `this.clock.tick(10);`